src/chocolatey/infrastructure.app/commands/ChocolateyRuleCommand.cs

```diff
@@ -0,0 +1,236 @@
+﻿// Copyright © 2017 - 2024 Chocolatey Software, Inc
+// Copyright © 2011 - 2017 RealDimensions Software, LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace chocolatey.infrastructure.app.commands
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using chocolatey.infrastructure.app.attributes;
+    using chocolatey.infrastructure.app.configuration;
+    using chocolatey.infrastructure.commandline;
+    using chocolatey.infrastructure.commands;
+    using chocolatey.infrastructure.logging;
+    using chocolatey.infrastructure.rules;
+    using chocolatey.infrastructure.services;
+
+    [CommandFor("rule", "view or list implemented package rules", Version = "2.3.0")]
+    [CommandFor("rules", "view or list implemented package rules (alias for rule)", Version = "2.3.0")]

```

Comment:
@AdmiringWorm this goes against where we want to go with simplifying the Chocolatey CLI command landscape.  We already have things like `choco source` and `choco sources`, but at the same time we don't have `choco pins` for `choco pin`.  Ultimately, likely in the next major release of Chocolatey CLI, we are going to start deprecating and removing these additional command aliases, so I am going to say that we don't make this problem worse by adding another alias, when it isn't needed.  Let's stick with `choco rule` as the command, and not have the alias.

@pauby adding for visibility on this decision.